.github/workflows/config.yml

```diff
@@ -71,9 +71,35 @@ jobs:
           path: coverage
           name: coverage
 
+  precomputed-buildstocks:
+    runs-on: ubuntu-latest
+    needs: [unit-tests]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Download and Install OpenStudio
+        run: |
+          wget -q https://github.com/NREL/OpenStudio/releases/download/v3.6.1/OpenStudio-3.6.1+bb9481519e-Ubuntu-20.04-x86_64.deb
+          sudo apt install -y ./OpenStudio-3.6.1+bb9481519e-Ubuntu-20.04-x86_64.deb
+          openstudio openstudio_version
+          which openstudio
+
+      - name: Generate precomputed buildstocks
+        run: |
+          pip install pandas
+          python test/update_yml_precomputed_files.py
+      
+      - name: Upload precomputed buildstocks
+        uses: actions/upload-artifact@v3
+        with:
+          path: test/tests_yml_files/yml_precomputed*/buildstock*.csv
+          name: precomputed_buildstocks
+
   analysis-tests:
     runs-on: ubuntu-latest
-    needs: [format-files]
+    needs: [precomputed-buildstocks]
```

Comment:
`unit_tests:project_integrity_checks` would be able to check that options_lookup is updated to match the housing characteristics, which otherwise would cause sampling to hang. This has happened before. 

I have tried to add precomputed-buildstocks into analysis-tests but the env set up for analysis-tests (with docker image) doesn't have python.

Open to other suggestions.

Reply:
One option is to use the openstudio CLI, which embeds both the ruby and python interpreters. You can call `openstudio labs foo.py` (the "labs" is because it's a bit experimental in the current OS release; in the upcoming release, it will be more mature and you can simply use `openstudio foo.py`).

Reply:
`open studio labs` doesn't support some of the python packages I need in the .py file. Is there any way to add that? @shorowit

Reply:
Oops, good point. I don't think there's an easy way to handle that, no.

But now I'm confused. You want to run python in `analysis-tests` -- it uses the OS docker image, which contains python, so what's the problem? You can just install pip and then pip install whatever you need. For example:
```
      - name: Install Python
        shell: bash
        run: |
          export DEBIAN_FRONTEND=noninteractive
          sudo apt update
          sudo -E apt-get install -y python3-pip
      - name: Install Python Dependencies
        run : |
          python3 -m pip install --progress-bar off --upgrade pip
          pip3 install --progress-bar off foobar
```